packages/kbn-data-stream-adapter/index.ts

```diff
@@ -9,13 +9,13 @@
 
 export { DataStreamAdapter } from './src/data_stream_adapter';
 export { DataStreamSpacesAdapter } from './src/data_stream_spaces_adapter';
-export { retryTransientEsErrors } from './src/retry_transient_es_errors';
-export { ecsFieldMap, type EcsFieldMap } from './src/field_maps/ecs_field_map';
+export { retryTransientEsErrors } from '@kbn/index-adapter/src/retry_transient_es_errors';
+export { ecsFieldMap, type EcsFieldMap } from '@kbn/index-adapter/src/field_maps/ecs_field_map';
```

Comment:
since `@kbn/index-dapter` is a package now can we import from it's entry file instead of internal paths
```suggestion
export { retryTransientEsErrors, ecsFieldMap, type EcsFieldMap } from '@kbn/index-adapter';
```

Reply:
Yup that's better, done!